### DISCUSSION TITLE
Verbindungsproblem mit dem CMT2300

### DISCUSSION THREAD

**Discussion by iukbox** [tbnobody/OpenDTU#947] [General]
Hallo

ich habe eine DTU mit beiden Modulen gebaut, es ist aber derzeit nur das 868 Mhz Funkmodul in Betrieb. Es soll mit einem HMT1800 kommunizieren.
Bei meinem WR bekomme ich keine Verbindung hin, wenn ich das beim WR des Nachbarn ( gleiches Modell ) probiere, dann funktioniert es und ich bekomme alle Werte angezeigt. Hat da jemand eine Idee?
Jemand meinte ich solle mal an der Frequenz 'rumspielen'.
![board_cmt2003](https://github.com/tbnobody/OpenDTU/assets/26631519/a019e4ee-436b-40f1-9f6c-3bcaa2115484)

**Comment by iukbox** [comment]
Keiner ne Idee?
Hardware getauscht, Antennen getauscht, gleiches Ergebnis. Beim Nachbarn am HMT 1800 gehts, bei mir nicht
@tbnobody   während der Entwicklung keine ähnlichen Probleme festgestellt? War evtl. ein Frequenzwechsel bei einigen Modellen nötig?

**Comment by tbnobody** [comment]
> @tbnobody während der Entwicklung keine ähnlichen Probleme festgestellt? War evtl. ein Frequenzwechsel bei einigen Modellen nötig?

Ne, da ist nichts bekannt. Die üblichen Dinge:

*  Eine zweite DTU am gleichen Inverter macht Probleme
* Je nachdem ob vorher schon eine andere DTU (am gleichen Tag) am WWechselrichter in betrieb war, kann es bis zu 15min dauern bis der Inverter wieder auf seine Standard Frequenz springt und von dort von der DTU wieder auf die Zielfrequenz (das ist die Frequenz die man einstellen kann in der Web GUI) gezogen wird.

**Comment by tugsi** [comment]
>  ![board_cmt2003](https://user-images.githubusercontent.com/26631519/239963873-a019e4ee-436b-40f1-9f6c-3bcaa2115484.jpg)

Der Beitrag ist zwar schon älter, aber darf ich fragen woher Du die PCB hast?

**Reply by iukbox** [reply]
Hab ich machen lassen.  Kannst dich gerne melden wenn du eins brauchst, hab noch Nachfolger hier liegen.

**Reply by Chr1k0** [reply]
Ich schließe mich mal an - Eventuell ne blöde Frage: Woher weiß ich, wie ich das CMT2300A an den ESP anschließen muss? Hier gibt es keinen Schaltplan :( https://ahoydtu.de/getting_started/  - nur für die Version mit NRF24. Auch steht dort ESP8266 würde nicht funktionieren. Du scheinst aber einen zu verwenden?

**Reply by iukbox** [reply]
Der Verdrahtungsplan war irgendwo auf GitHub. Danach hab ich die Platine entworfen. 
Du hast nen Link zur ahoyDTU eingestellt, wir sind bei der OpenDTU.

**Comment by LoneRanger4711** [comment]
@iukbox, ist das Problem mittlerweile gelöst? Wenn nein, dann könnte es sein, dass Du eine WiFi-Variante des Hoymiles bekommen hast. Der funkt bereits im 2,4GHz im Gegensatz zu der Sub-1G Variante. Das CMT2300A-Modul ist für den ISM-Bereich von 868MHz. Welche Antenne hast Du denn am CMT2300A-Modul angeschlossen?

**Reply by iukbox** [reply]
So 50:50 gelöst. Der WR wird jetzt angezeigt aber die Reaktionszeiten bzw. Aktualisierungsintervalle sind grottenschlecht. Eine dynamische  Regelung wäre wohl  nicht möglich auf Grund der Zeiten.
Da ich mir ( bedingt durch den Umbau meiner Anlage ) nur noch die Daten anzeigen lasse reicht mir das so.
Liegt aber wahrscheinlich am WR da ich mehrere OpenDTU mit der Platine gebaut hab die an anderen WR problemlos funktionieren.

**Comment by tugsi** [comment]
> Hab ich machen lassen. Kannst dich gerne melden wenn du eins brauchst, hab noch Nachfolger hier liegen.

Danke, bin auf der Suche nach dem PCB auf den FusionBoard gestossen und dies jetzt geordert, aber Danke  für die Info und  das Angebot.

**Comment by huste511** [comment]
Die Frage nach der Antenne ist schon mal ein guter Hinweis.
Weiterhin mal die Sendeleistung kontrollieren und ggf. in Schritten auf max. 14dBm stellen und jeweils eine Zeit lang abwarten.
DTU möglichst nah zu dem MWR schaffen ... und auch noch mal Seriennummer genau 1:1 vergleichen.
